spec/index.bs

```diff
@@ -698,15 +714,19 @@ initially empty.
 
     1. Set |localName| to |tagName| in [=ASCII lowercase=].
     1. Set |attribute| to |attribute| in [=ASCII lowercase=].
+    1. If |attribute| starts with "on", then return {{TrustedScript}}.
```

Comment:
There's probably a better way of doing this? The question is do we care? If someone does `getAttributeType('div', 'one')` and it says TrustedScript is that an issue?

Possibly could do this via https://html.spec.whatwg.org/#event-handler-content-attributes some lookup based on this concept?

Reply:
Fwiw Chrome currently does something more safisticated than just looking for the on prefix.

Reply:
Correct, we should only be blocking on actual event handler attributes.

Reply:
Fixed this

Reply:
This also addresses #474